packages/worker/src/sdk/users/users.ts

```diff
@@ -340,8 +332,10 @@ export const bulkCreate = async (
 
   for (const newUser of newUsersRequested) {
     if (
-      newUsers.find((x: any) => x.email === newUser.email) ||
-      existingEmails.includes(newUser.email)
+      newUsers.find(
+        (x: any) => x.email.toLowerCase() === newUser.email.toLowerCase()
+      ) ||
+      existingEmails.includes(newUser.email.toLowerCase())
```

Comment:
The lowercasing should be applied to all three look ups for 
- `getExistingTenantUsers`
- `getExistingPlatformUsers`
- `getExistingAccounts`

We should be sure that the `emails: string[]` passed to each of those functions should do 
`emails = emails.map(e => e.toLowerCase)`. It's a bit tedious to do it in each but safest in case they would ever be called elsewhere. 

We would also want to make sure the emails returned from `searchExistingEmails` are lowercased as the comparison here on 338 is comparing the requested email with the email returned from the underlying document.